backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -45,6 +45,7 @@ public class TestEvent extends BaseTestInfo {
   @JoinColumn(name = "test_order_id")
   private TestOrder order;
 
+  @JsonIgnore
   @OneToMany(mappedBy = "testEvent", cascade = CascadeType.MERGE, fetch = FetchType.LAZY)
```

Comment:
This is a good callout - I honestly don't remember why I set it up this way, because we pretty much always care about the results on a TestEvent. It doesn't seem to have any effect if I remove this `FetchType=LAZY`, so I'm going to address that in a separate PR.